db/seeds.rb

```diff
@@ -1,4 +1,4 @@
-require 'factory_bot_rails'
+rquire 'factory_bot_rails'
```

Comment:
I think this may have been an accidental change, and may be linked to the aformentioned schema changes.

Reply:
Total accident, unrelated